src/SerializableAsync.ts

```diff
@@ -160,10 +218,24 @@ export class SerializableAsync extends SerializableBase implements ISerializable
         return await this.postFrom(realObj)
     }
 
+    /**
+     * `preFrom` can be overwritten to manipulate the value before the parsing.
+     * This allows to add logic to the deserialization without having to override the fromAny method.
+     *
+     * @param value the object that will be manipulated before the actual parsing.
+     * @returns tht manipulated object or a promise that resolves in the manipulated object
+     */
     protected static preFrom(value: any): Promise<any> | any {
         return value
     }
 
+    /**
+     * `postFrom` can be overwritten to manipulate the value after the parsing.
+     * This allows to add logic to the deserialization without having to override the fromAny method.
```

Comment:
This allows to add logic to the **parsing** without having to override the fromAny method.